Running `make` shows error missing `#include <cstring>` in file `strencodings.h`

I just tried to `make` btcdeb and I got the following error: 

```
make[1]: Entering directory '/home/standup/btcdeb'
  CXX      util/libbitcoin_a-strencodings.o
In file included from util/strencodings.cpp:6:
./util/strencodings.h: In function ‘bool ValidAsCString(const string&)’:
./util/strencodings.h:31:26: error: ‘strlen’ was not declared in this scope
   31 |     return str.size() == strlen(str.c_str());
      |                          ^~~~~~
./util/strencodings.h:17:1: note: ‘strlen’ is defined in header ‘<cstring>’; did you forget to ‘#include <cstring>’?
   16 | #include <iterator>
  +++ |+#include <cstring>
   17 | #include <string>
make[1]: *** [Makefile:1345: util/libbitcoin_a-strencodings.o] Error 1
make[1]: Leaving directory '/home/standup/btcdeb'
make: *** [Makefile:1626: all-recursive] Error 1

```

After this I simple included `#include <cstring>` in the file `./util/strencodings.h` and I ran fine.

Please fix :)

--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
-AC_PREREQ([2.71])
+AC_PREREQ([2.60])
 define(_CLIENT_VERSION_MAJOR, 0)
 define(_CLIENT_VERSION_MINOR, 4)
 define(_CLIENT_VERSION_REVISION, 22)

--- a/util/strencodings.h
+++ b/util/strencodings.h
@@ -17,7 +17,7 @@
 #include <string>
 #include <vector>
 
-// #include <cstring>
+#include <cstring>
 #include <functional>
 
 #define ARRAYLEN(array)     (sizeof(array)/sizeof((array)[0]))

